### PR TITLE
docs: fabric-mod documentation

### LINKED ISSUE #72: Add documentation skeleton

### PR DESCRIPTION
**Pull request by talwinder50**
Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>
Closes #72

### CODE REVIEW

**Comment by talwinder50** [comment]
verify-unittest

**Comment by talwinder50** [comment]
verify-unitest

File: docs/source/index.rst [hunk 1]
```diff
@@ -36,6 +36,7 @@ modularity and versatility for a broad set of industry use cases.
    releases
    questions
    status
+   gobackextension
```
Comment (rolsonquadras) [inline docs/source/index.rst]: Is this a UI label or invisible UI element ? If this is a visible label, then we need to have better name.

Comment (talwinder50) [reply docs/source/index.rst]: No this is not visibile element. The purpose of this rst file is to create that link from original fabric documentation to fabric mod documentation.

**Pull request merged**